### DISCUSSION TITLE
Specifying initial concentrations

### DISCUSSION THREAD

**Discussion by yoerga** [phydrus/phydrus#53] [Q&A]
When specifying initial concentrations in a profile, what units should concentrations be in, and are there options (i.e. mass of solute per volume of water vs. mass of solute per volume of soil)? In the Hydrus GUI, you have the option of either and need to specify.

**Comment by martinvonk** [comment]
When initializing the `phydrus.Model()` you can specify the argument `mass_units`. That defaults to `'mmol'` but you can choose any mass unit as long as it's available in HYDRUS-1D.

**Reply by yoerga** [reply]
Thank you for your response. I think my question was poorly phrased. We are wondering if the concentration term for the initial conditions represents the mass of solute per volume of water or the mass of solute per volume of soil and whether it is possible to select either option.

**Reply by martinvonk** [reply]
We use the old version of the [HYDRUS-1D Manual](https://github.com/phydrus/source_code/blob/main/HYDRUS1D-4.08.pdf) version 4.08 (2009) since that belongs to the [FORTRAN source code](https://www.pc-progress.com/en/Default.aspx?H1). In this manual there is **no** option to specify the initial concentration as portion of the volume of soil (which is documented in manual 4.17 (2013) as _lMassIni_ in BLOCK F of SELECTOR.IN). So therefore I'd say that the initial concentration has to be set as **mass per volume of transport domain**.

Best thing to do (to be 100% sure) is to check using the example notebooks. [Notebook 8](https://github.com/phydrus/phydrus/blob/master/examples/hydrus_orig/Example_8%20-%20hydrus%20tutorial.ipynb) has solute transport and you can try both options and see what matches the outcome as documented in the tutorial.
